tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_SimpleWenoImpl.cpp

```diff
@@ -265,9 +266,24 @@ void test_simple_weno_1d(const std::unordered_set<Direction<1>>&
                            neighbor_modified_vars);
 }
 
-void test_simple_weno_2d(const std::unordered_set<Direction<2>>&
-                             directions_of_external_boundaries = {}) noexcept {
+void test_simple_weno_1d() noexcept {
+  INFO("Test simple_weno_impl in 1D");
+  const auto gl = Spectral::Quadrature::GaussLobatto;
+  const auto gauss = Spectral::Quadrature::Gauss;
+
+  test_simple_weno_1d_impl(gl);
+  test_simple_weno_1d_impl(gauss);
+
+  // Test with particular boundaries labeled as external
+  test_simple_weno_1d_impl(gl, {{Direction<1>::lower_xi()}});
```

Comment:
maybe either indicate that gauss points don't work in this case or test gauss points too?

Reply:
This is for checking that the sum over neighbors correctly doesn't try to use missing neighbors (at external boundaries). This is unrelated to the choice of quadrature.

Perhaps my use of the word "boundaries" in the comment is misleading? The thing that really matters, in the simple weno implementation, is the loop over data received from the neighbors, which may or may not be one per direction. So if you think it would be helpful, I can rephrase this accordingly.

Reply:
Ah okay, yes I think adding to the comment something like "This is independent of the quadrature, since we only verify that we don't try to use a non-existent neighbor"

Reply:
Sounds good to me 👍

I'm planning a followup PR where I clean up a few inconsistencies and improve comments in various places in the limiter code... I'd slightly prefer to make this change in the followup PR, but if you like I'll do it here now. Let me know what you think.

Reply:
Sounds good to me :)